### PR TITLE
zh-translation:/docs/reference/glossary/adapters.md

### PR DESCRIPTION
**Pull request by vflong**
ref: [servicemesher/istio-official-translation/issues/1094](https://github.com/servicemesher/istio-official-translation/issues/1094)

### CODE REVIEW

File: content/zh/docs/reference/glossary/adapters.md [hunk 1]
```diff
@@ -2,10 +2,6 @@
...
 
-Adapters are plug-ins to [Mixer](#mixer), Istio's [policy and telemetry](/docs/reference/config/policy-and-telemetry/) component, which enable it to interface
-with an open-ended set of infrastructure backends that deliver core functionality, such as logging,
-monitoring, quotas, ACL checking, and more.
-The exact set of adapters used at runtime is determined through configuration and can easily be
-extended to target new or custom infrastructure backends.
+适配器（Adapters）是 Istio [策略和遥测](/docs/reference/config/policy-and-telemetry/)组件 [Mixer](#mixer) 的插件, 可使其与一组开放式基础架构后端交互，这些后端可提供核心功能，例如日志记录、监控、配额、ACL 检查等等。运行时所使用的精确的适配器集合是通过配置确定的，并可以针对新的或定制的基础架构后端轻松扩展。
 
-[Learn more about adapters](/docs/reference/config/policy-and-telemetry/adapters).
+[了解更多关于适配器的信息](/docs/reference/config/policy-and-telemetry/adapters)。
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/adapters.md]: 请在链接中增加中文前缀 `/zh`。

File: content/zh/docs/reference/glossary/adapters.md [hunk 2]
```diff
@@ -2,10 +2,6 @@
 title: Adapters
```
Comment (rootsongjc) [inline content/zh/docs/reference/glossary/adapters.md]: 标题需要翻译。

Comment (vflong) [reply content/zh/docs/reference/glossary/adapters.md]: 由于[此页面](https://github.com/servicemesher/istio-official-translation/issues/1074)引用了 `adapters` 术语，导致术语标题无法修改，所以需要先翻译[此页面](https://github.com/servicemesher/istio-official-translation/issues/1074)，然后重新提交。
报错信息如下：
```bash
$ make serve                                                           
Building with the build container: gcr.io/istio-testing/build-tools:2019-10-24T14-05-17.
Building sites … ERROR 2019/11/04 23:12:03 Could not find glossary entry for 'adapters' ("/work/content/zh/about/contribute/creating-and-editing-pages/index.md:551:12")
ERROR 2019/11/04 23:12:03 Could not find glossary entry for 'adapters' ("/work/content/zh/about/contribute/creating-and-editing-pages/index.md:561:15")
Total in 150454 ms
Error: Error building site: logged 2 error(s)
Makefile.core.mk:30: recipe for target 'serve' failed
make: *** [serve] Error 255
Makefile:104: recipe for target 'serve' failed
make: *** [serve] Error 2
```

**Pull request closed without merging**